app/client/components/billing/billing.tsx

```diff
@@ -127,7 +127,8 @@ const BillingRenderer = ([mdaResponse, invoiceResponse]: [
                       ...invoice,
                       pdfPath: `/api/${invoice.pdfPath}`,
                       currency: paidPlan.currency,
-                      currencyISO: paidPlan.currencyISO
+                      currencyISO: paidPlan.currencyISO,
+                      productTitle: specificProductType.urlPart
```

Comment:
if you're going to use `urlPart`, either call it 
```suggestion
                      productUrlPart: specificProductType.urlPart
```
...OR something like...
```suggestion
                      productCode: specificProductType.urlPart
```